src/translations/fr.json

```diff
@@ -217,5 +217,15 @@
     "StopComputation": "Arrêter le calcul",
     "ColumnsList": "Liste des colonnes",
     "LabelSelectList": "Configurer colonnes",
-    "CheckAll": "Sélectionner tout / aucun"
+    "CheckAll": "Sélectionner tout / aucun",
+
+    "LockoutLine": "Consigner la ligne",
+    "TripLine": "Déclencher la ligne",
+    "EnergiseOnOneEnd": "Mettre sous tension à vide depuis {substation}",
```

Comment:
({substation}) instead ? like in english translation

Reply:
This concerns the following PR :
https://github.com/gridsuite/gridstudy-app/pull/320
@niconoir